queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT name, escape_attempts FROM animals WHERE animals.weight_kg > 10.5;
 SELECT * FROM animals WHERE animals.neutered = true;
 SELECT * FROM animals WHERE animals.name != 'Gabumon';
 SELECT * FROM animals WHERE animals.weight_kg >= 10.4 AND animals.weight_kg <= 17.3;
+
+START TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals WHERE animals.date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(escape_attempts) FROM animals WHERE animals.escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) as total_escape_attempts FROM animals GROUP BY neutered;
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
```

Comment:
- [ ] 👍 Your query is functioning correctly and provides the minimum and maximum weight from all animals. However, the project requirement asks for a query that shows the minimum and maximum weight for each type of animal, which your current query doesn't fulfill. Please consider updating your code to display the minimum and maximum weights for each animal type. This will ensure that your query aligns with the project requirements. Great effort so far! 👏📊
- [ ] Please also update the screenshot in the PR description if necessary!